service-worker.js

```diff
@@ -23,117 +23,191 @@
  */
 'use strict';
 
-self.addEventListener('install', function(event) {
+var CACHE = 'kiwixjs-cache';
```

Comment:
This should be renamed to indicate that it's a cache of assets, used by the SW

Reply:
`kiwixjs-assetCache`? NB The Cache API could perfectly well be used in jQuery mode. The fact that we only implement it in SW mode is an arbitrary decision (albeit a sensible one for us at this moment in time). Cache API is available in `window`, so accessible from app.js.

Reply:
All right. This name looks fine

Reply:
I've changed the name. Be sure to delete the old Cache, either before downloading the commit, or in dev tools.